source/elements/oneDNN/source/api/primitives/rnn.rst

```diff
@@ -0,0 +1,16 @@
+..
+  Copyright 2019 Intel Corporation
+
+.. default-domain:: cpp
+
+RNN
+~~~
+
```

Comment:
Is this missing a [blurb](https://intel.github.io/mkl-dnn/group__dnnl__api__rnn.html#details) and an 'API' sub-section?

Reply:
There is no blurb in the header file (at least not where the `dnnl_api_rnn` group starts). Would you like to provide a sentence? (Or we could use one from the developer reference.)

Reply:
Fixed. Please review.